.github/workflows/CD.yml

```diff
@@ -83,17 +87,21 @@ jobs:
       uses: appleboy/ssh-action@master
       env:
         SITE_URL: ${{ env.KAYUNGA_SITE_URL }}
+        KAYUNGA_TAG_ID: "G-8PKFYRVCTG"
```

Comment:
Could you add these from secrets as well? Also, confirm if you are able to set secrets now. Sending you DB_CONNECTION_STRINGs for both so you can have them set up as well, and we should be good to go.

Reply:
> Could you add these from secrets as well? Also, confirm if you are able to set secrets now. Sending you DB_CONNECTION_STRINGs for both so you can have them set up as well, and we should be good to go.

Disregard, actually we should instead be removing this. I was confusing it with the API. My bad.